Podcast/Assets.xcassets/logo.imageset/Contents.json

```diff
@@ -1,23 +1,23 @@
 {
-  "info" : {
-    "author" : "zeplin",
-    "version" : 1
-  },
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "group4.png",
+      "filename" : "bitmap.jpg",
```

Comment:
Are we updating the app icon to this too?

Reply:
yes im going to add the app icons to this PR!